configs/llm-router/aws.yaml

```diff
@@ -0,0 +1,19 @@
+head_node_type:
+  name: head
+  instance_type: g5.48xlarge
+  resources:
+    cpu: 8
+
+auto_select_worker_config: true
```

Comment:
can you delete everything from this line below? none of it should be needed for single node